frontend/src/app/shared/components/attachments/attachment-list/attachment-list.component.ts

```diff
@@ -32,67 +32,27 @@ import {
   Input,
   OnInit,
 } from '@angular/core';
-import { map, tap } from 'rxjs/operators';
-import { Observable } from 'rxjs';
-import { HalResource } from 'core-app/features/hal/resources/hal-resource';
 import { IAttachment } from 'core-app/core/state/attachments/attachment.model';
-import { TimezoneService } from 'core-app/core/datetime/timezone.service';
 import { UntilDestroyedMixin } from 'core-app/shared/helpers/angular/until-destroyed.mixin';
 import { AttachmentsResourceService } from 'core-app/core/state/attachments/attachments.service';
-import isNewResource from 'core-app/features/hal/helpers/is-new-resource';
 
 @Component({
   selector: 'op-attachment-list',
-  templateUrl: './attachment-list.html',
+  templateUrl: './attachment-list.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class AttachmentListComponent extends UntilDestroyedMixin implements OnInit {
-  @Input() public resource:HalResource;
+export class OpAttachmentListComponent extends UntilDestroyedMixin implements OnInit {
```

Comment:
kudos for integrating this component into the now common `OpAttachmentComponent`! 👍🏽